lamindb/_feature.py

```diff
@@ -94,7 +113,9 @@ def categoricals_from_df(df: pd.DataFrame) -> dict:
     for key in string_cols:
         c = pd.Categorical(df[key])
         if len(c.categories) < len(c):
-            categoricals[key] = c
+            logger.warning(
+                "consider changing the dtype of string column `key` to categorical"
```

Comment:
Right?
```suggestion
               f "consider changing the dtype of string column {key} to categorical"
```

Reply:
Yes, sorry, didn't push my latest fixes... 😔 -- it's already fixed locally.